### ISSUE TITLE
Question: Some VPN  servers blocked by notrack

### ISSUE THREAD

**Issue by jovendor** [quidsup/notrack#220]
Hi,

I have no track set up with Raspberry PI working well however when I use some VPN servers it seem to block them... I checked "Domain blocking" and I can see country is not blocked... How can I unblock them?

Much appreciated

**Comment by mintyfreshpenguin** [comment]
Did you get this working?

**Comment by joelven** [comment]
No sure... I set my router with public dns numbers and all devices with the notrack ip.  Is this set up correct?

**Comment by mintyfreshpenguin** [comment]
I don’t think it matters what DNS servers you put into the router. So-long as your devices are using NoTrack’s IP for DNS, no queries should reach the router. IP’s set in the router would only be used when your devices use the router for DNS.  

> In fact if I set the raspberry pi "IP number" into the DNS box of my router it does not work.

When you set the RPI's IP in the router was it under the router's DHCP setting? This is how mine is set up. Not all router will let you set the DNS for DHCP though.
